src/components/Dialogs/index.jsx

```diff
@@ -29,38 +28,64 @@ const MessageItem = (props) => {
   return <div className={styles.message}>{text}</div>;
 };
 
-const areaRef = createRef();
+// const areaRef = createRef();
 
-const onClickHandler = () => {
-  const areaValue = areaRef.current?.value;
-  alert(areaValue);
-};
+// const onClickHandler = () => {
+//   const areaValue = areaRef.current?.value;
+//   alert(areaValue);
+// };
+
+// const onChangeHandler = (event) => {
+//   console.log(event.target.value);
+// };
 
-const onChangeHandler = (event) => {
-  console.log(event.target.value);
+let onSendMessageClick = () => {
+  store.dispatch(sendMessageCreator());
+};
+let onNewMessageChange = (event) => {
+  let body = event.target.value;
+  store.dispatch(updateNewMessageBodyCreator(body));
 };
 
 const Dialogs = ({ list }) => {
   return (
     <div className={styles.dialogs}>
       <div className={styles.dialogsItems}>
         <DialogItem
-          name={dialogsData.map((dia, index) => (
-            <li key={index}>{dia.name}</li>
+          name={store._state.DIALOGS.CONVERSATIONS.map((dia, index) => (
+            <li className={styles.link} key={index}>
+              <img
+                className={styles.image}
+                src="https://mir-s3-cdn-cf.behance.net/project_modules/disp/ea7a3c32163929.567197ac70bda.png"
+                alt=""
```

Comment:
alt can`t be empty, check the console for the errors. All warnings/errors should be resolved before PR will be merged to the master

Reply:
Added users_logo subscription

Reply:
The console is without errors